lib/translator/common.h

```diff
@@ -23,6 +27,14 @@ void DebugLog(const char* format, ...);
 
 void SetDebugCallback(void (*callback)(const char*));
 
+template <typename T>
+bool ReadValue(absl::Span<const uint8_t> data, T& out) {
+  static_assert(std::is_pod_v<T>, "only supports POD types");
```

Comment:
The `type_traits` header should be included explicitly for `is_pod_v`, even though it happens to work anyway by coincidence for now.